transaction-gossip.md

```diff
@@ -97,7 +102,12 @@ The proof **must** show that:
 
 ### Gossip Rules
 
-Nodes should OFFER transactions to the DHT nodes in their routing table.  A DHT node should only be offer'd a transaction that is inside of its radius.
+Nodes should OFFER transactions to the DHT nodes sourced from their secondary routing table.
+
+A DHT node should only be offer'd a transaction that is inside of its radius.
```

Comment:
I know this is preliminary so far.

Do we gossiped to all nodes in the routing tables? Or do we just gossip to some random one?

Reply:
I'm thinking that we define a constant `N` and and that nodes should gossip to up-to `N` nodes that satisfy `node.radius >= self.radius` and up-to `N` nodes that satisfy `node.radius < self.radius` to push the message out in both directions.